packages/adaptive-ui-designer-figma/src/cli/string-utils.ts

```diff
@@ -0,0 +1,18 @@
+// https://stackoverflow.com/questions/36658624/how-do-i-convert-a-string-to-spinal-case-in-javasscript
+export function spinalCase(str: string) {
```

Comment:
I've been using `change-case` in other packages. More than we need for one format. :shrug:

Reply:
Oh, looks like it's already in this library. They call it `kebab`.

Reply:
Good call, refactored to `kebabCase`